src/metabase/api/card.clj

```diff
@@ -415,6 +418,46 @@
       :else
       nil)))
 
+(defn- card-is-verified?
+  "Return true if card is verified, false otherwise. Assumes that moderation reviews are ordered so that the most recent
+  is the first. This is the case from the hydration function for moderation_reviews."
+  [card]
+  (-> card :moderation_reviews first :status #{"verified"} boolean))
+
+(defn- changed?
+  "Return whether there were any changes other than ignored values.
+
+  returns false because changes to collection_id are ignored:
+  (changed? {:collection_id 1 :description \"foo\"}
+            {:collection_id 2}
+            {:ignore #{:collection_id}})
```

Comment:
We don't have Marginalia or Cljdoc set up rn but if we did (and we probably should get one of them set up at some point) it will format docstrings as Markdown. So it's good to add a newline and then indent code examples in docstrings four spaces so they'd be interpreted as code blocks